content/en/docs/concepts/policy/limit-range.md

```diff
@@ -80,11 +80,11 @@ This object defines minimum and maximum Memory/CPU limits,  default cpu/Memory r
 Create the `limit-mem-cpu-per-container` LimitRange in the `limitrange-demo` namespace with the following kubectl command:
```

Comment:
the text "in the `limitrange-demo` namespace" will need to be modified if the namespace is removed.